clarity/src/vm/docs/mod.rs

```diff
@@ -279,6 +279,93 @@ Note: This function is only available starting with Stacks 2.1.",
 "#,
 };
 
+const PRINCIPAL_PARSE_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(principal-parse principal-address)",
```

Comment:
The name of this function seems backwards to me -- I usually think of parsing as "buffer-to-structure", but this is kind of "structure-to-buffers" -- maybe a better name is `principal-destruct`? This also conveys that it is the inverse of `principal-construct`.

Reply:
Fixed